daffodil-lib/src/main/scala/org/apache/daffodil/exceptions/ThrowsSDE.scala

```diff
@@ -57,7 +57,8 @@ trait ThrowsSDE {
 
   final def schemaDefinitionError(str: String, args: Any*): Nothing = SDE(str, args: _*) // long form synonym
```

Comment:
Above at line 85, missed deprecating another form of SDW call without warnID. Need to find where these are all called also and add more warnIDs. 

Why this review tool won't let me put a comment on an unchanged line of a file.... is annoying!